components/infobox/wikis/worldoftanks/infobox_map_custom.lua

```diff
@@ -49,8 +49,10 @@ function CustomInjector:parse(id, widgets)
 	elseif id == 'custom' then
 		return Array.append(widgets,
 			Cell{name = 'Map Season', content = {args.season}},
-			Cell{name = 'Size', content = {(args.width or '') .. 'x' .. (args.height or '')}},
-			Cell{name = 'Battle Tier', content = {(args.btmin or '') .. '-' .. (args.btmax or '')}},
+			Cell{name = 'Size', content = {(args.width or '') .. ' x ' .. (args.height or '')}},
+			Cell{name = 'Battle Tier', content = {String.isEmpty(args.btmin) and
```

Comment:
the logic here (plus next line) seems faulty
and potentially will cause errors

Reply:
Not sure where the faulty logic is.

Reply:
isEmpty(a) and isEmpty(b) and '' or (a..b)
will error if one of both is nil
you want
isNotEmpty(a) and isNotEmpty(b) and (a..b) or nil